recipe/meta.yaml

```diff
@@ -1,18 +1,13 @@
 {% set chost = macos_machine %}
-# only reset build_number to 0 if _all_ output versions are increased as well
-{% set build_number = 3 %}
-{% set build_offset = 100 %}
-# use build offset to keep our builds newer than old libgfortran builds from another feedstock
-{% set final_build_number = build_number + build_offset %}
 
 {% if gfortran_version is undefined %}
 {% set gfortran_version = "11.3.0" %}
-{% set libgfortran_version = "5.0.0" %}
+{% set libgfortran_soversion = "5" %}
 {% endif %}
 
-{% set libgfortran_major_version = libgfortran_version.split(".")[0] %}
-{% set libgfortran_next_major = (libgfortran_version|int +1)|string %}
-{% set max_libgfortran_version = libgfortran_next_major + ".0.0.a0" %}
+# only reset build_number to 0 if _all_ output versions are increased as well
+# otherwise increment and keep above 100
```

Comment:
Why do we need to keep it above 100?

Reply:
This comes from @minrk's previous PR: https://github.com/conda-forge/gfortran_impl_osx-64-feedstock/pull/87

Maybe it no longer applies here? I thought it did but I could be wrong!

Reply:
Since we are changing `libgfortran` version, it doesn't apply anymore.

Reply:
Ah ok. Then ignore me and I will back out my change. Sorry for the noise!